File: a004. 文文的求婚/Java/Lyu.java
import java.util.Scanner;

public class Lyu {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int year;
        while (scan.hasNext()) {
            year = scan.nextInt();
            System.out.println((year % 4 == 0 && year % 100 != 0 || year % 400 == 0) ? "閏年" : "平年");
        }
        scan.close();
    }
}

File: a004. 文文的求婚/Python/Lyu.py
symbolYear = {True: '閏年', False: '平年'}
while(True):
    try:
        year = int(input())
        print(symbolYear[(year % 4 == 0 and year % 100 != 0) or year % 400 == 0])  
    except:
        break
